lib/microos.pm

```diff
@@ -43,8 +43,7 @@ sub microos_reboot {
     # No grub bootloader on xen-pv
     # grub2 needle is unreliable (stalls during timeout) - poo#28648
     assert_screen 'grub2', 300;
-    send_key('ret') if match_has_tag('grub2');
-
+    send_key('ret') if get_var('DISABLE_GRUB_TIMEOUT');
```

Comment:
this can impact some places... this will need some VRs in several places of O3 and OSD.